Components/Transceivers/RDMA/RDMA Linux RT x64.lvproj

```diff
@@ -431,8 +435,11 @@ AddOutputFilter chunkFilter
 				<Property Name="Destination[1].destName" Type="Str">Support Directory</Property>
 				<Property Name="Destination[1].path" Type="Path">/Components</Property>
 				<Property Name="Destination[1].path.type" Type="Str">&lt;none&gt;</Property>
-				<Property Name="DestinationCount" Type="Int">2</Property>
-				<Property Name="Source[0].itemID" Type="Str">{143C9DB3-6F08-483B-8C9A-420653A3ACD1}</Property>
+				<Property Name="Destination[2].destName" Type="Str">Errors</Property>
```

Comment:
This needs to be in the Windows project instead of the Linux project. The `Post-Build Action.vi` that copies the errors files to the correct location is looking for output in `Windows\Components\Errors`.

Reply:
Now it should be correct.